src/translations/strings.yaml

```diff
@@ -928,6 +928,20 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+helpSheets: 
+  dnsTitle:
+    value: Custom DNS settings
+    comment: Title label for the custom dns help sheet
+  dnsHeader: 
+    value: What is custom DNS?
+    comment: Header label for the custom dns help sheet
+  dnsBody1:
+    value: Whenever you connect to a website, a DNS (domain name system) first turns the domain name (e.g. www.mozilla.org) into an IP address that allows your internet traffic to reach that destination.
```

Comment:
Just checking if this content has been approved by UX? The help text doesn't seem to explain what 'resolve websistes' means in the screen UI.

Reply:
@traceyvantyghem

Reply:
+1 @vinocher . We don't seem to be answering the question from dnsHeader 'What is a _custom_ DNS?').

Reply:
@MattLichtenstein I agree that updating the title makes sense (per Flod's suggestion). 

I didn't work on this content (and don't know much about the subject matter), so I checked in with Chris, but he didn't have any additional updates. Chris is returning to content lead on VPN, so I guess we can leave the body content as-is.